src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+::The Gutter - Bathroom - Mirror
+(set: $mirrorSeen to true)
+I gripped the edges of the counter a bit tighter. My face looked unfamiliar. The stranger’s eyes were weary, like they’d been hung with weights.
+
+//“Hey. $playerName," the mirror stranger said, "You look pretty fucked up. Like somebody dosed you.”//
+
+(click: "dosed")[$dissolve["Yeah, well. How about we skip the obvious and you give me a goddamn hand here?"]]
+
+(click: "give me a goddamn hand")[$dissolve[//“A hand? HA! That’s richer than the fuckos in those nice trailers,"// The face got even nastier, contorting and screaming at me like a high wind.
+
+//"How about you stop sobbing in the potty and find your friend? Are you gonna crumble this time you no-good bitch? You fucking idiot! How are you gonna handle when it all comes down? When the rubber hits the road?”//]]
+
+(click: "rubber hits the road")[$dissolve[My jaw slacked and my lip quivered. I held my hand up and watched it tremble for a moment. But then, like Jay-sus settling the bubbles in a shook up bottle of Rock N Rye, my muscles went calm.]]
+
+(click: "calm")[$dissolve[I fixed myself into a battle stance and roared back, “I handle fine. I handle better! I ride steady and smooth like a ‘65 Cutlass at midnight with the headlights off and I’m COMING]]
+
+(click: "COMING")[THROUGH]
+
+(click: "THROUGH")[YOUR]
+
+(click: "YOUR")[FUCKING]
+
+(click: "FUCKING")[B U L L S H I T]
+
+(click: "B U L L S H I T")[$dissolve[I heard a crunching at the edge of my perception.]]
+
+(click: "crunching")[$dissolve[I looked down to see the sink starting to shift out of the wall. I had to scramble to keep it in place.(if: $sinkPuke is true)[ A bit of my puke sloshed onto my hand from the sink. Ugh.] I hadn’t realized how hard I’d been gripping it.]]
+
+(click: "gripping it")[$dissolve[The mirror stranger got me riled. She sure was a bitch, but she’s dead on. Funny how she’s fucking right as can be.
+
+(display: "The Gutter - Bathroom - Menu")]]
+
+::The Gutter - Bathroom - Toilet flashback
+(set: $toiletflashbackSeen to true)
+I sat down for a second. Just needed to clear my head. But, like everything else tonight, what I wanted didn’t happen. My ass pancaked onto the flimsy, plastic seat while my brain slipped beyond me, into the past.
+
+(click: "into the past")[$dissolve[I’m eleven years old. Anna and I are dancing while my mom watches from a lawn chair. It’s a summer day when it feels like you’re racing around a band of the sunshine that’s never gonna flatten out and end.]]
+
+(click: "flatten out and end")[$dissolve[Anna tugs me over to this group of guys that she’s been talking to. Leo and his dad, some others. We take our boombox over and do this dance routine we’ve been practicing for a few days, nothing fancy, but we really worked at it and it’s kinda long. There are a few parts where we pick each other up.]]
```

Comment:
The last sentence here — "There are a few parts where we pick each other up" — is a little confusing. What does this tell us? Are they falling down intentionally or accidentally? I think this could use a little more contextualization.

Reply:
I don't think it's that anybody's falling down, more that one will lift up the other ice-dancing style.

Reply:
Nathan's got the right idea. It's picking people up like in a cheer routine. But, having them both pick each other up is a weird way to phrase it.

I rewrote as "There's even a part where Anna lifts me."